Set env path

Tacha-S opened the pull request:
<!--
Please remove the appropriate section.
For example, if this is a new feature, remove all sections except for the "New feature" section

If this is your first time opening a PR, be sure to check the contribution guide:
https://gazebosim.org/docs/all/contributing
-->

# 🦟 Bug fix

Fixes #630 

## Summary
<!-- Describe your fix, including an explanation of how to reproduce the bug
before and after the PR.-->
I set the environment variables to correctly load plugins when launching with a composable node.

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

Tacha-S committed:
Set env path

Signed-off-by: Tatsuro Sakaguchi <tatsuro.sakaguchi@g.softbank.co.jp>

azeey reviewed (commented):
Thanks for the contribution!!
 
This approach works, but only when using the python launch file. It wouldn't work for those using the `gzserver` action from XML. So, could you move these changes to the action (https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_sim/ros_gz_sim/actions/gzserver.py)?

Amronos commented:
I think this PR fixes #630 instead of #549.

Tacha-S committed:
Move env settings to gzserver action

Signed-off-by: Tatsuro Sakaguchi <tatsuro.sakaguchi@g.softbank.co.jp>

Tacha-S commented:
> This approach works, but only when using the python launch file. It wouldn't work for those using the gzserver action from XML. So, could you move these changes to the action ([ros2/ros_gz_sim/ros_gz_sim/actions/gzserver.py](https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_sim/ros_gz_sim/actions/gzserver.py))?

I moved to setting environment variables in [ros2/ros_gz_sim/ros_gz_sim/actions/gzserver.py](https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_sim/ros_gz_sim/actions/gzserver.py).

> I think this PR fixes https://github.com/gazebosim/ros_gz/issues/630 instead of https://github.com/gazebosim/ros_gz/issues/549.

Oops, it seems I accidentally selected the one above when choosing an issue.
Thank you. I’ve fixed it.

ahcorde reviewed (changes requested):
Do you mind to add some documentation about this ?

Tacha-S committed:
Add document

azeey reviewed (approved):
LGTM!

ahcorde commented:
https://github.com/Mergifyio backport jazzy
